Added DemoneCeleste

ReLoia opened the pull request:
Added the extension for www.demoneceleste.it

Checklist:

- [x] Updated `extVersionCode` value in `build.gradle` for individual extensions
- [ ] Updated `overrideVersionCode` or `baseVersionCode` as needed for all multisrc extensions
- [ ] Referenced all related issues in the PR body (e.g. "Closes #xyz")
- [ ] Added the `isNsfw = true` flag in `build.gradle` when appropriate
- [ ] Have not changed source names
- [ ] Have explicitly kept the `id` if a source's name or language were changed
- [x] Have tested the modifications by compiling and running the extension through Android Studio

alessandrojean commented:
Please use the tool mentioned in the [Contributing guide](https://github.com/tachiyomiorg/tachiyomi-extensions/blob/master/CONTRIBUTING.md#tools) to generate the icons, the one currently in the PR is using a different pattern from the other extensions.

stevenyomi commented:
Please do not credit by Discord ID in code comment. Please remove that line in your next commit to this PR. If you want to credit, please add the following line to your next commit message:
```
Co-authored-by: stevenyomi <95685115+stevenyomi@users.noreply.github.com>
```
(generated by https://coauthoredby.netlify.app/)

ReLoia commented:
The last commit fixed the problems mentioned in your comments. I changed the icons of the extensions following the Contributing guide (somehow I did already make the icon using the right tool but I may have uploaded the files from the wrong .zip folder) and removed credit by Discord ID present in the first commit. I hope now the PR is fine and will be accepted.
